models.js

```diff
@@ -34,4 +32,21 @@ exports.selectArticleById = (article_id) => {
         
       });
       
-  };
\ No newline at end of file
+  };
+
+  exports.selectArticles = () => {
+    return db.query (
+        `
+    SELECT articles.*, COUNT(comments.comment_id) AS comment_count
+    FROM articles
+    LEFT JOIN comments ON articles.article_id = comments.article_id
+    GROUP BY articles.article_id
+    ORDER BY articles.created_at DESC;
+  `)
+        .then(({rows}) =>{
+            return rows;
+        })
+        .catch((error) => {
```

Comment:
You don't need a catch here. If you want to console log the error you can do so in the catch in the controller.